js/misc/paygManager.js

```diff
@@ -43,6 +43,22 @@ const EOS_PAYG_IFACE = '<node> \
 </interface> \
 </node>';
 
+var PaygErrorDomain = GLib.quark_from_string('payg-manager-error');
```

Comment:
Nitpick: I think the variable name (`PaygErrorDomain`) should match the quark string `(`payg-manager-error`) to avoid any possible confusion in future. So this should have a quark string of `payg-error`, not `payg-manager-error`.

Reply:
Ok